package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "vso-android-extension",
-  "version": "2.164.1",
+  "version": "1.168.0",
```

Comment:
Why was the major version downgraded here?

Reply:
So what should the version be now here?

Reply:
A newer version shouldn't ever lower the 'major' version.  The 'minor' is always the sprint.  If you're making potentially incompatible changes with an earlier version, then you'll need to bump the 'major'.

| Original | Update
|--------|------------
| 2.161.1 | 2.168.0
| 3.164.1 | 3.168.0
| 1.5.0 (example) | 2.168.0 (when major changes) |